doc/source/dev/roadmap-detailed.rst

```diff
@@ -427,7 +427,13 @@ The following improvements will help SciPy better serve this role.
     - nested ANOVA
     - analysis of covariance (ANCOVA)
 
-- Add additional tools for meta-analysis; currently we have just `combine_pvalues`.
+  Also, provide an infrastructure for implementing hypothesis tests.
+- Add additional tools for meta-analysis (e.g. the Benjamini–Hochberg and
+  Benjamini–Yekutieli procedures)
+- Add tools for survival analysis
+- Replace inverse transform sampling with a more suitable method of random
+  variate sampling where appropriate
```

Comment:
Talking about UNU.RAN there? It also sounds like the proposal of Christoph (we could add a note on that too)

Reply:
Yes, I put this in here for Christoph.